test/app.spec.js

```diff
@@ -123,6 +123,25 @@ describe('app', () => {
     });
   });
 
+  describe('start', () => {
+    beforeEach(() => {
+      sinon.stub(messenger.app, 'listen');
```

Comment:
We converted the _Lenses_ project over to `sandbox` and it's under the _didn't get to that yet_ here. I'll create an issue since there are other test files that would need this conversion also.